### DISCUSSION TITLE
How to get maddyctl?

### DISCUSSION THREAD

**Discussion by lolwierd** [emersion/maddy#533] [Q&A] [answered]
I tried tar xvf on the tarball downloaded from github and also tried to build maddy on my machine. But I did not get the maddyctl executable anywhere. Where can I find it?

**Comment by foxcpp** [comment, marked as answer]
Probably I forgot to update documentation somewhere. Since 0.6, maddyctl is part of maddy executable.
That is, `maddyctl creds list` is `maddy creds list`.
